service/ml/model_registry_usage_test.go

```diff
@@ -58,20 +58,20 @@ func ExampleModelRegistryAPI_CreateComment_modelVersionComments() {
 
 }
 
-func ExampleModelRegistryAPI_CreateModel_models() {
```

Comment:
These changes are unrelated but have come up few times before. These should be harmless.